Library/brew.rb

```diff
@@ -58,6 +58,10 @@ def require?(path)
   ENV["PATH"] += "#{File::PATH_SEPARATOR}#{HOMEBREW_ENV_PATH}/scm"
 
   if cmd
+    if cmd == "cask" && (HOMEBREW_CELLAR/"brew-cask").exist?
+      system(HOMEBREW_BREW_FILE, "uninstall", "--force", "brew-cask")
+    end
```

Comment:
> Suggested line?

Here: https://github.com/Homebrew/brew/blob/b53f0c5ada753d5ded6f6fbe4f920a4a43696394/Library/brew.rb#L80

> My concern was removing it before we'd required any `internal_cmd`.

Why would that be an issue?